docs/release_notes.md

```diff
@@ -1,5 +1,30 @@
 # Release Notes
 
+## Version 1.3.0
+
+### Core
+
+- Implemented a new solver for fat simulation.
+- Implemented a new constraint to preserve the internal shape of volumetric geometries.
+- Implemented a new constraint to preserve the volume in tetrahedralized geometries.
+- Extended all existing solvers to support stiffness overrides per constraint type.
+
+### Maya
+
+- Added AdnFat deformer.
+- Added stiffness overrides per constraint type to all deformers.
+- Improved UX to prevent informative dialogs to prompt.
+- Improved UX to prevent dialogs to prompt when running commands from script.
+
+### Bug Fixes
+
+- Fixed a bug that was redistributing the masses per point in very specific cases and only if the painted masses map was not the default one.
+
+### Known Limitations
+
+- Interaction muscle-to-muscle is not supported. *AdonisFX-1211*
+- Exporter and Importer do not support locators and sensors. *AdonisFX-1215*
```

Comment:
For your info, we have removed the following lines from the previous version without being complieted:

```
- Slide on geometry constraints in Fast mode are limited to simple setups. They need to run in Quality mode to ensure stability in complex scenarios. *AdonisFX-1213*
- Volume ratio gain effect is dependent to the total volume of the muscle. This might require to use volume ratios with a range different to the default [0, 2] if needed. *AdonisFX-1137*
```

This is because those two issues have been reviewed with Marco and we have to run investigation before we can clearly say that they are limitations.